src/libopensc/card-myeid.c

```diff
@@ -131,9 +135,13 @@ static int myeid_init(struct sc_card *card)
 	u8 defatr[SC_MAX_ATR_SIZE];
 	size_t len = sizeof(defatr);
 	const char *atrp = myeid_atrs[MYEID_INFINEON_CHIP_ATR];
-
+	
 	LOG_FUNC_CALLED(card->ctx);
+
+	card->name = myeid_card_name;
```

Comment:
Yes I should get more familiar with git. I did "git merge upstream/master" before making my changes and believed that this syncs my code with the master, but apparently that is not enough.Please understand that with this PR I was trying to quickly fix something that was not working at all, before moving on to another project. I am willing to correct all of the details but currently have very limited time to allocate for OpenSC dev so it might take time.